e2e/basic/assert_cleanups.go

```diff
@@ -0,0 +1,42 @@
+package basic
+
+import (
+	"os"
+	"testing"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+// assertCleanupInstance is a helper function that cleans up a single instance.
+func assertCleanupInstance(t *testing.T, instance *knuu.Instance) error {
```

Comment:
A nice one. 

[optional] we can add something more generic batch destroy function to the knuu pkg that receives a series of instances e.g:
```go
func BatchDestroy(instance ...*Instance) error {
...
}
```

it might help to have a cleaner test code.